dxp/aui_script.md

```diff
@@ -0,0 +1,148 @@
+		{
+			closeable: true,
+			message: 'I like drink tea while eating french potatoes',
+			type: 'success'
+		}
+	).render('#myId');
+</aui:script>
+```
+
+The output will be a code like this:
+
+```js
+AUI().use('liferay-alert', function (A) {
+	(function() {var $ = AUI.$;var _ = AUI._;
+		new Liferay.Alert(
+			{
+				closeable: true,
+				message: 'I like drink tea while eating french potatoes',
+				type: 'success'
+			}
+		).render('#myId');
+	})()
+}
+```
+
+Note in the AlloyUI components, coming from the alloy-ui repository, classes are made available from the global variable `A` on the scope of the `aui:script`. Here as in this [code snippet the Menu component is defined](https://github.com/liferay/alloy-ui/blob/master/src/aui-menu/js/aui-menu.js#L39]).
+
+However, components made available from the `frontend-js-aui-web` module will be under the global Liferay variable. Like this [example here, `Liferay.Upload`](https://github.com/liferay/liferay-portal/blob/815f48f484351e18b61e4b9c9fbf40f0609bdc56/modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/upload.js#L1541), used in the code example above.
+
+#### Import a ES module using `require` property
+
+Allows users to require a modern ES module as in the following example:
+
+Considering We already have `metal-clipboard` and `metal-dom` dependencies listed [on our module package.json](https://help.liferay.com/hc/en-us/articles/360018159771-The-Structure-of-OSGi-Bundles-Containing-npm-Packages-)
+
+By default the name of the loaded resource will be the transformed path to camel-case default exported module/file:
+
+```jsp
+<aui:script require="metal-clipboard/src/Clipboard">
+    // Note We have the provided path to require metal-clipboard/src/Clipboard normalized and camel-case as a variable name
+	new metalClipboardSrcClipboard.default();
+</aui:script>
+```
+
+Also, We can set an alias with the `as` word after the provided path. It will be look like:
+
+```jsp
+<aui:script require="my-custom-lib/src/path/main as myLib">
+	// ... code that use some function from myLib like `myLib.someFunction();`
+</aui:script>
+```
+
+The output will be:
+
+```js
+Liferay.Loader.require('my-custom-lib/src/path/main', function(myCustomLibSrcPathMain) {
+	try {
+		(function() {
+			var myLib = myCustomLibSrcPathMain;
+			var $ = AUI.$;var _ = AUI._;
+			// ... code that use some function from myLib like `myLib.someFunction();`
+		})();
+	} catch (err) {
+		console.error(err);
+	}
+}
+```
+
+You can find more information on [this link](https://help.liferay.com/hc/en-us/articles/360017882752-Loading-Modules-with-AUI-Script-in-Liferay-DXP#loading-es2015-and-metaljs-modules)
+
+#### Using `position` to control script placement
+
+This property will change the way in which / how the script will be injected into the page. By default, those scripts that are wrapped by `aui:script` will be added to the minified files as described [here](#inline-js-on-the-jsp).
```

Comment:
I would remove `/ how`

Reply:
I was probably thinking in portuguese when writing this part

Reply:
Yeah, sounded more like Spanish than English :sweat_smile: